File: src/main/java/com/rs/java/game/area/multi/WaterbirthDungeonMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class WaterbirthDungeonMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Waterbirth Dungeon Multi";
	}

	public WaterbirthDungeonMulti() {
		super(9886, 10142, 7236, 7492, 7748, 11589);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/WaterbirthDungeonMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class WaterbirthDungeonMulti : Area(9886, 10142, 7236, 7492, 7748, 11589) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Waterbirth Dungeon Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
